features/activity/cancel_abandon/README.md

```diff
@@ -12,4 +12,7 @@ cancelled
 * When the SDK requests cancel of an abandon mode activity, the workflow code
   immediately unblocks, the activity result being cancelled.
 * Nothing is sent to the server, the activity worker never is notified of the
-  cancellation attempt
\ No newline at end of file
+  cancellation attempt
+
+
+**NOTE:** This feature does not exist in the Go SDK
```

Comment:
I fully acknowledge this is a bit anal, but maybe add a "as of <date>" next to this line, since in theory everyone should _eventually_ be conformant

Reply:
Sure, or "currently" or whatever.

Reply:
Updated to clarify